src/main/java/com/google/sps/servlets/AvailabilityServlet.java

```diff
@@ -14,14 +14,39 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.Tutor;
+import com.google.sps.data.TimeRange;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.Optional;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.*; // Can be deleted after hardcoded database is removed
+
 
 @WebServlet("/availability")
 public class AvailabilityServlet extends HttpServlet {
+    private static final int TIME_0800AM = TimeRange.getTimeInMinutes(8, 00);
```

Comment:
I think these would be good to add in the SampleData class

Reply:
I agree, I'll add this as a new commit.